### DISCUSSION TITLE
Celery Unlock Chord Retries

### DISCUSSION THREAD

**Discussion by tnolan-stratxai** [celery/celery#9154] [Q&A]
Hi there,

I have some tasks that I run in a `Group` where at least two of them are long running tasks i.e. might take 20/30 minutes to complete. When monitoring things on Flower I can see that Celery is executing a Chord Unlock multiple times, leading to very large numbers of tasks being executed on some of the workers.
<img width="1398" alt="Screenshot 2024-07-26 at 09 37 04" src="https://github.com/user-attachments/assets/96a50990-48a0-4aa3-9d68-d4e7d843f74b">

I'm wondering is there a way to delay the Chord Unlock retries here for a certain group of tasks if I know it's going to take at least X amount of time? Or is there potentially a different way I should be handling something like this?

Cheers.

**Comment by thedrow** [comment]
I think you're looking for `result-chord-retry-interval` but I'm not sure since the documentation on it is scarce.
See [docs](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-chord-retry-interval).
